src/test/json2xlsx.js

```diff
@@ -82,51 +82,83 @@ describe('json2xlsx', () => {
       const sheets = XLSX.readFile('./temp/ccd-definitions.xlsx').Sheets;
       assert(Object.keys(sheets).length > 0, 'No sheets have been created');
 
+      const assertCell = (sheetName, cell, expectedCellValue) => {
+        const actualCellValue = expectedCellValue === undefined ? sheets[sheetName][cell] : sheets[sheetName][cell].v;
+        assert.equal(actualCellValue, expectedCellValue, `Unexpected value found in ${cell} cell of ${sheetName} sheet`);
+      };
+
       sheetNames.forEach(sheetName => {
         assert(sheets[sheetName], `No sheet corresponding to JSON file ${sheetName} exists`);
         if (sheetName === 'AuthorisationCaseEvent') {
```

Comment:
If you are updating the tests thoughts on these sequences of ifs being changed to a switch?